open_prices/proofs/migrations/0007_pricetag.py

```diff
@@ -0,0 +1,117 @@
+# Generated by Django 5.1.4 on 2024-12-16 14:04
+
+import django.contrib.postgres.fields
+import django.db.models.deletion
+import django.utils.timezone
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("prices", "0004_price_type"),
+        ("proofs", "0006_add_proof_prediction_table"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="PriceTag",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "created",
+                    models.DateTimeField(
+                        default=django.utils.timezone.now,
+                        verbose_name="When the tag was created in DB",
```

Comment:
use [help_text](https://docs.djangoproject.com/en/5.1/ref/models/fields/#help-text) for these kind of descriptions, and make the verbose_name short (or leave it empty)

Reply:
Indeed, I should have checked better Copilot suggestion ;)